roles/sap_ha_set_netweaver/README.md

```diff
@@ -1,19 +1,29 @@
 # sap_ha_set_netweaver Ansible Role
 
 Ansible role for SAP NW High Availability Setup
-
 ## Scope
 
 - **RedHat Enterprise Linux**
-    - Tested on RHEL 8.2
+    - Tested on RHEL 8.2, 8.4 and later
+    - Please also check 
+    - [Overview of the Red Hat Enterprise Linux for SAP Solutions subscription](https://access.redhat.com/solutions/3082481)
+    - [Automating SAP HANA Scale-Up System Replication using the RHEL HA Add-On](https://access.redhat.com/articles/3004101)
 
-- **Azure** 
+- **Azure**
     - Tested
     - Followed the steps based on the guide published in
-        - [Azure HA Guide](https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/sap/high-availability-guide-rhel-netapp-files)
+        - [Azure Pacemaker Setup Guide](https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/sap/high-availability-guide-rhel-pacemaker)
 
 - **AWS**
```

Comment:
Would suggest defining the IaaS type, e.g. `, EC2 Instance`

Also to control scope, add the `AWS Classic environment` is out of scope?